components/config/caching.rst

```diff
@@ -1,26 +1,27 @@
 .. index::
    single: Config; Caching based on resources
 
-Caching Based on Resources
+Caching based on Resources
```

Comment:
Looked odd to me at first, looks ok to me now - and I verified that lowercase makes sense from one quick source. Mostly, I don't care either way :)